apps/admin/next.config.js

```diff
@@ -1,3 +1,9 @@
+import million from 'million/compiler';
+
+const millionConfig = {
+  auto: true,
```

Comment:
```suggestion
  auto: { rsc: true },
```

Reply:
@aidenybai what is RSC project btw?

Reply:
nextjs app router uses React Server Components(RSC)

Reply:
Ah okay, Got it 👍🏻.